graftm/unpack_sequences.py

```diff
@@ -54,10 +88,17 @@ def _guess_sequence_type_from_string(self, seq):
                 num_nucleotide += 1
             elif residue in aas:
                 num_protein += 1
+            elif residue == '-':
+                if self.gapped_alignment:
+                    continue
+                else:
+                    logging.warning("Input sequences are gapped ('-' detected in sequences). These gaps will be removed, but the resultant reads may not be identical to the originals (if insertions and deletions have been removed, etc..)")
```

Comment:
Maybe `Input sequences are gapped ('-' detected in sequences) suggesting they are aligned. Resultant reads may not be identical to the original aligned reads if insertions and deletions have previously been removed`